Main.cpp

```diff
@@ -946,21 +948,21 @@ bool PlayIntroTitle()
 
     // Load the Intro slides
     BITMAP **apIntroSlides = new BITMAP *[SLIDECOUNT];
-    ContentFile introSlideFile("Base.rte/Title/IntroSlideA.bmp");
```

Comment:
I don't really think the title stuff should be in GUI, since it's literally not a user interface. I'd suggest making a separate top level folder called Cinematics or something.